SomerenUI/EditDrinkForm.cs

```diff
@@ -29,20 +29,22 @@ private void btnCreateDrink_Click(object sender, EventArgs e)
             bool isAlcoholic = rdoTrue.Checked;
             try
             {
-                if (txtDrinkName.Text.IsNullOrEmpty())                
-                    throw new ArgumentNullException();                
+                if (txtDrinkName.Text.IsNullOrEmpty())
+                    throw new ArgumentNullException();
+
+                double price = double.Parse(txtDrinkPrice.Text);
+                int stock = int.Parse(txtDrinkStock.Text);
 
-                Drink drink = new Drink(double.Parse(txtDrinkPrice.Text), isAlcoholic, txtDrinkName.Text, int.Parse(txtDrinkStock.Text));
+                Drink drink = new Drink(price, isAlcoholic, txtDrinkName.Text, stock);
                 drinkService.CreateDrink(drink);
-                ShowMessage(Properties.Resources.SuccessfullyAdded, drink.Name);
-                Close();
+                ShowMessage(Properties.Resources.SuccessfullyAdded, drink.Name, this);
             }
             catch (Exception ex)
             {
-                string errorMessage = ex is ArgumentNullException ? Properties.Resources.NullField : Properties.Resources.ErrorMessage;
+                string errorMessage = ex is ArgumentNullException ? Properties.Resources.ErrorMessageNullField : Properties.Resources.ErrorMessage;
```

Comment:
The error message can be used only for an empty Name field: "Need to input a name. Error: " but the naming "ErrorMessageNullField" suggests that I can use it for any NullField errors. Please, make it clearer like "ErrorMessageNullNameField".